assets/js/modules/datastorage.js

```diff
@@ -0,0 +1,13 @@
+// store
+export default class keepdata {
+  static addToLocalStorage(library) {
+    const storage = localStorage.setItem('inputVal', JSON.stringify(library));
+    return storage;
+  }
+
+  static getStorage() {
+    const storage = localStorage.getItem('inputVal') == null
+      ? [] : JSON.parse(localStorage.getItem('inputVal'));
+    return storage;
+  }
```

Comment:
- [ ] @Mukaba kindly consider using the arrow syntax in your methods, as required by the project instructions
![image](https://user-images.githubusercontent.com/61408860/228285680-a3f49546-827e-4988-baf0-9e2218600161.png)